content/applications/general/companies/digest_emails.rst

```diff
@@ -5,10 +5,10 @@ Digest emails
 *Digest Emails* are periodic snapshots sent via email to users in an organization that include
 high-level information about how the business is performing.
 
-To start sending digest emails, begin by navigating to :menuselection:`Settings app --> Statistics
+To start sending digest emails, begin by navigating to :menuselection:`Settings app --> Emails
 section`, activate the :guilabel:`Digest Emails` feature, and click :guilabel:`Save`.
 
-.. image:: digest_emails/digest-email-settings.png
+.. image:: digest_emails/digest-email-settings-screenshot.png
    :align: center
```

Comment:
Can remove all instances of `align:center` in the file, as it's an obsolete image parameter
```suggestion
```